{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/plugin.py

```diff
@@ -49,14 +49,14 @@
 ########################################
 
 # To add a custom initialization task, create a bash script template under:
-# {{ cookiecutter.module_name }}/templates/{{ cookiecutter.plugin_name }}/jobs/init/
+# {{ cookiecutter.module_name }}/templates/{{ cookiecutter.plugin_name }}/tasks/
 # and then add it to the MY_INIT_TASKS list. Each task is in the format:
 # ("<service>", ("<path>", "<to>", "<script>", "<template>"))
 MY_INIT_TASKS: list[tuple[str, tuple[str, ...]]] = [
     # For example, to add LMS initialization steps, you could add the script template at:
-    # {{ cookiecutter.module_name }}/templates/{{ cookiecutter.plugin_name }}/jobs/init/lms.sh
+    # {{ cookiecutter.module_name }}/templates/{{ cookiecutter.plugin_name }}/tasks/{{ cookiecutter.plugin_name }}/init.sh
     # And then add the line:
-    ### ("lms", ("{{ cookiecutter.plugin_name }}", "jobs", "init", "lms.sh")),
+    ### ("lms", ("{{ cookiecutter.plugin_name }}", "tasks", "{{ cookiecutter.plugin_name }}", "init.sh")),
```

Comment:
You're right. Do you have any particular service name in mind for this one?
also, should we drop the `.sh` extension for the job? all of the official plugins use an `init` file.

Reply:
@codewithemad Let's use `lms`, because that's the most common service in which plugins will need to execute tasks.

> also, should we drop the .sh extension for the job? all of the official plugins use an init file.

Good question. I lean dropping the `.sh` extension since that's what the official plugins do, but I'll leave it up to your discretion.

Reply:
I like the `.sh` extension because it makes it easier to edit the files in the IDE.

Reply:
@CodeWithEmad , could you apply this change?

Reply:
Oh, I totally forgot about this. Absolutely.

Reply:
can you take a look now.